Nimda/Core/CommandContainer.php

```diff
@@ -47,14 +47,14 @@ public function loadPlugins()
      *
      * @param array $plugins
      */
-    private function loadCorePlugins(array $plugins)
+    private function loadCoreCommands(array $plugins)
```

Comment:
Since everything else is being changed with this refactor, change variable names also